backend/tests/e2e/conftest.py

```
@@ -0,0 +1,471 @@
+    # Temporarily patch settings for database initialization
+    temp_settings = type('Settings', (), {})()
+    temp_settings.history_database_url = db_url
+    temp_settings.history_enabled = True
+    
+    # Patch the database manager to use our isolated database
+    isolated_db_manager = DatabaseManager(temp_settings)
+    isolated_db_manager.engine = engine
+    isolated_db_manager.SessionLocal = lambda: Session(engine)
+    
+    with patch('tarsy.config.settings.get_settings', return_value=temp_settings), \
+         patch('tarsy.repositories.base_repository.DatabaseManager', return_value=isolated_db_manager):
```

Ruff (0.12.2)

Line too long (103 > 88)

(E501)